core-graphics/src/display.rs

```diff
@@ -141,6 +141,90 @@ impl CGDisplay {
         CGDisplay::new(kCGNullDirectDisplayID)
     }
 
+    /// Return the number of online displays with bounds that include the
+    /// specified point.
+    #[inline]
+    pub fn display_count_with_point(point: CGPoint) -> Result<u32, CGError> {
+        let mut matching_display_count: u32 = 0;
+        let result = unsafe {
+            CGGetDisplaysWithPoint(point, 0, ptr::null_mut(), &mut matching_display_count)
+        };
+        if result == 0 {
```

Comment:
Would you mind comparing against `kCGErrorSuccess` in these cases?

Reply:
Done (and rebased on latest `main`).